services/net/syndication/SyndicationAction.cs

```diff
@@ -85,52 +85,38 @@ private async Task ImportFeedAsync(IDataSourceIngestManager dataSource, Syndicat
         {
             try
             {
-                // Fetch content body separately if required
-                if (bool.Parse(dataSource.DataSource.GetConnectionValue("fetchContent")))
```

Comment:
This should not be done for each item every time.  It should only be done if it's a new piece of content, or needs to be updated.